CHANGELOG.md

```diff
@@ -1,4 +1,89 @@
-## 2.19.4 (Unreleased)
+## 2.19.5 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.4**
+- Supported Terraform version: **v1.0**
+
+### Features:
+#### Accounts
+1. Implemented support for AWSChina in **aviatrix_account** and data source with the following new attributes:
+- ``awschina_role_app``
+- ``awschina_role_ec2``
+2. Implemented support for AzureChina in **aviatrix_account** and data source with the following new attributes:
+- ``azurechina_subscription_id``
+- ``azurechina_directory_id``
+- ``azurechina_application_id``
+- ``azurechina_application_key``
+3. Implemented support for AWSTS in **aviatrix_account** and data source with the following new attributes:
+- ``awsts_account_number``
+- ``awsts_cap_url``
+- ``awsts_cap_agency``
+- ``awsts_cap_mission``
+- ``awsts_cap_role_name``
+- ``awsts_cap_cert``
+- ``awsts_cap_cert_key``
+- ``awsts_ca_chain_cert``
+4. Implemented support for AWSS in **aviatrix_account** and data source with the following new attributes:
+- ``awss_account_number``
+- ``awss_cap_url``
+- ``awss_cap_agency``
+- ``awss_cap_account_name``
+- ``awss_cap_role_name``
+- ``awss_cap_cert``
+- ``awss_cap_cert_key``
+- ``awss_ca_chain_cert``
+
+#### Firewall Network
+1. Implemented support for Fail Close and Network List Excluded From East-West Inspection in **aviatrix_firenet**
+2. Implemented support for colon character in ``tags`` in **aviatrix_firewall_instance**
+
+#### Gateway
+1. Implemented support for Applying Route Entry in **aviatrix_gateway**
+2. Implemented support for AWS Top Secret provider in **aviatrix_gateway**
+3. Implemented support for AWS Secret provider in **aviatrix_gateway**
+4. Implemented support for configuring gateway keepalive settings
+- **aviatrix_controller_gateway_keepalive_config**
+5. Implemented support for colon character in ``tags`` in **aviatrix_gateway**
```

Comment:
Colons were already supported in tags before this. This change should not have any changes from the user's perspective so I'm not sure we need this in the changes.